### PR TITLE
ci: trigger ci on push specific branches

### PR DESCRIPTION
**Pull request by liya2017**
### What problem does this PR solve?

ci: trigger ci on push specific branches

### Check List 

Tests 

- tests

### Release note 

```release-note
None: Exclude this PR from the release note.
```

### CODE REVIEW

File: .github/workflows/ci_benchmarks_macos.yaml [hunk 1]
```diff
@@ -19,6 +19,14 @@ env:
...
   prologue:
     name: prologue
+    if: |
+      contains(github.event_name, 'pull_request')
+      || contains('refs/heads/master,
+                      refs/heads/trying,
+                      refs/heads/staging', github.ref)
+      || (github.ref == 'refs/heads/develop' && github.event.head_commit.author.name != 'bors[bot]')
+      || startsWith(github.ref, 'refs/heads/rc/')
+      || (github.repository_owner != 'nervosnetwork')
```
Comment (keroro520) [inline .github/workflows/ci_benchmarks_macos.yaml:29]: btw, I prefer to replace `contains` with the equal mark `=`. I try to refactor this block to make it more readable:

```yaml
if: |
    github.repository_owner != 'nervosnetwork' ||
    githu.event_name != 'push' ||
    (
        githu.event_name == 'push' &&
        (
            github.ref == 'refs/heads/master' ||
            github.ref == 'refs/heads/trying' ||
            github.ref == 'refs/heads/staging' ||
            (github.ref == 'refs/heads/develop' && github.event.head_commit.author.name != 'bors[bot]') ||
            startsWith(github.ref, 'refs/heads/rc/')
        )
    )
```

**Review by keroro520 — APPROVED** [review]

File: .github/workflows/ci_benchmarks_macos.yaml (hunk 1 above)
Comment (liya2017) [reply .github/workflows/ci_benchmarks_macos.yaml:29]: yes,you are right, thanks for your remanding.

**Review by quake — APPROVED** [review]

**Comment by quake** [comment]
bors r=quake,keroro520

**Review by doitian — APPROVED** [review]

**Review by zhangsoledad — APPROVED** [review]

**Comment by zhangsoledad** [comment]
bors r=quake,keroro520,doitian,zhangsoledad

**Pull request merged**